gocloak.go

```diff
@@ -5,6 +5,10 @@ import (
 	"gopkg.in/resty.v1"
 )
 
+// Version number of gocloak
+// exported so implementations can include the version number in debug output
+const Version = "3.5.1"
```

Comment:
It can complicate the creation of new tags. Now i can push new tag without admin rights in repos, but with this changes only @Nerzal will be able to create a new version, since only he can push changes to master branch or we will need to create a PR before pushing new tag.
I'm, OK with this changes, but we need to define a workflow of bumping a new version.

Reply:
We might think of another solution for this :D